scripts/build/build.ps1

```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference="Stop"
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-_ "$RepoRoot\scripts\build\generate-version.ps1"
+. "$RepoRoot\scripts\build\generate-version.ps1"
```

Comment:
Ah makes sense now. Took that comment out because I started looking it up. 

The `_` function is pretty neat